src/test/regress/expected/create_index_optimizer.out

```diff
@@ -3160,3 +3160,70 @@ explain (costs off)
  Optimizer: Pivotal Optimizer (GPORCA) version 3.1.0
 (4 rows)
 
+--
+-- Check spgist index work with AO tables
+--
+-- insert to AO table with spgist index
+create table spgist_ao_test(idx_col int4range)
```

Comment:
In this file all other SQL words are in capital (**CREATE** but not **create**).
Should not it be consistent with the rest of the file?